py/client/pydeephaven/agg.py

```diff
@@ -0,0 +1,363 @@
+#
+#   Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+#
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import List, Union, Sequence
+
+from pydeephaven.proto import table_pb2
+
+_GrpcAggregation = table_pb2.Aggregation
+_GrpcAggregationColumns = _GrpcAggregation.AggregationColumns
+_GrpcAggregationCount = _GrpcAggregation.AggregationCount
+_GrpcAggregationPartition = _GrpcAggregation.AggregationPartition
+_GrpcAggSpec = table_pb2.AggSpec
+
+
+def _to_sequence(cols: Union[str, List[str]]) -> Sequence[str]:
+    if isinstance(cols, str):
+        return (cols,)
+    else:
+        return cols
```

Comment:
I think that the code is in the server library.  You should make a common library for the client containing the code, since it will be used all over the place.

Reply:
remove first, will address it in #3558